source/octf/analytics/statistics/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ PRIVATE
     ${CMAKE_CURRENT_LIST_DIR}/IoStatisticsSet.h
     ${CMAKE_CURRENT_LIST_DIR}/WorksetCalculator.h
     ${CMAKE_CURRENT_LIST_DIR}/WorksetCalculator.cpp
```

Comment:
Maybe we should remove the old class? Or make WorksetCalculatorDevices inherit from WorksetCalculator so to not duplicate code.

Reply:
I think, that's ok design. Inherit won't work because methods have different arguments. And in fact we don't repeat implementation. In device version, we take the calculator for specific drive and pass call further. So we use very well encapsulation here.